test/k8sT/manifests/test-verifier.yaml

```diff
@@ -38,3 +38,4 @@ spec:
   - key: "node.kubernetes.io/unreachable"
     operator: "Exists"
   hostNetwork: true
+  nodeName: k8s1
```

Comment:
None of the manifests have the node name specified. Would it make sense to think on a solution that doesn't have infrastructure details in the tests?

Reply:
At the very least we should use `cilium.io/ci=k8s1` label

Reply:
We already indirectly encode an infrastructure detail in the test. We currently need to mount the Cilium sources inside the `test-verifier` container. That means we can only run this test when the sources are available inside the VM (no GKE).

And I just found out that the second flake affecting `test-verifier` (cf. https://github.com/cilium/cilium/issues/14792) only happens when the pod is scheduled on k8s2. If it wasn't for that second flake, we could use a DaemonSet I guess.

Reply:
Unfortunately switching to the use of labels to match the node broke the fix for the scheduling issue. See https://github.com/cilium/cilium/issues/14791#issuecomment-788176744. https://github.com/cilium/cilium/pull/15156 should have the new fix, needed only on v1.8.